docs/APDB.md

```diff
@@ -0,0 +1,47 @@
+Alert Production Database (APDB) Schema
+=======================================
+
+The [apdb](./python/lsst/sdm_schemas/schemas/apdb.yaml) schema describes the Alert Production Database (APDB).
+This database is updated by the [Alert Production Pipeline](https://github.com/lsst/ap_pipe) and used for nightly [ap_verify](https://github.com/lsst/ap_verify) runs within continuous integration.
+Previous processing runs may differ from the current schema.
+The user-queryable Prompt Products Database (PPDB) is expected to have a very similar schema to the APDB.
```

Comment:
I don't think we need to say anything about PPDB here, that one is still more or less unknown w.r.t. schema management.